<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigitRecognitionClass</class>
 <widget class="QMainWindow" name="DigitRecognitionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DigitRecognition</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="RealTime">
       <attribute name="title">
        <string>RealTime</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QComboBox" name="deviceBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>21</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopStartButton">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="cameraWindow">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>640</width>
                <height>480</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>640</width>
                <height>480</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="invertedGrayscaledWindow">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>320</width>
                  <height>240</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>320</width>
                  <height>240</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="subImagesWindow">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>320</width>
                  <height>240</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>320</width>
                  <height>240</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="aiCheckBox">
                 <property name="text">
                  <string>AI</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <property name="topMargin">
                  <number>80</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>10</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Rate Chance</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="rateSpin">
                   <property name="decimals">
                    <number>2</number>
                   </property>
                   <property name="minimum">
                    <double>0.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.020000000000000</double>
                   </property>
                   <property name="value">
                    <double>0.940000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="topMargin">
                  <number>70</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Black</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="blackSpin">
                   <property name="maximum">
                    <number>255</number>
                   </property>
                   <property name="value">
                    <number>255</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <property name="topMargin">
                  <number>70</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>White</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="whiteSpin">
                   <property name="maximum">
                    <number>255</number>
                   </property>
                   <property name="value">
                    <number>255</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="topMargin">
                  <number>70</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Condition</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="conditionSpin">
                   <property name="maximum">
                    <number>255</number>
                   </property>
                   <property name="value">
                    <number>160</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="topMargin">
                  <number>70</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Step</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="stepSpin">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>255</number>
                   </property>
                   <property name="singleStep">
                    <number>1</number>
                   </property>
                   <property name="value">
                    <number>255</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <property name="topMargin">
                  <number>70</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Min</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="minSpin">
                   <property name="minimum">
                    <number>10000</number>
                   </property>
                   <property name="maximum">
                    <number>10000000</number>
                   </property>
                   <property name="singleStep">
                    <number>10000</number>
                   </property>
                   <property name="value">
                    <number>1010000</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <property name="topMargin">
                  <number>70</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Max</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="maxSpin">
                   <property name="minimum">
                    <number>100000</number>
                   </property>
                   <property name="maximum">
                    <number>10000000</number>
                   </property>
                   <property name="singleStep">
                    <number>10000</number>
                   </property>
                   <property name="value">
                    <number>10000000</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <property name="topMargin">
                <number>70</number>
               </property>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Size</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="digitSizeSpin">
                 <property name="minimum">
                  <number>28</number>
                 </property>
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                 <property name="singleStep">
                  <number>28</number>
                 </property>
                 <property name="value">
                  <number>336</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Draw">
       <attribute name="title">
        <string>Draw</string>
       </attribute>
       <widget class="DrawWindow" name="drawWindow">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>565</width>
          <height>452</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>565</width>
          <height>452</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>565</width>
          <height>452</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="clearButton">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QPushButton" name="recognizeButton">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Recognize</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultWindow">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
font: 75 60pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawWindow</class>
   <extends>QLabel</extends>
   <header>drawwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DigitRecognition.qrc"/>
 </resources>
 <connections/>
</ui>
